### PR TITLE
add dear doge testnet

### PR DESCRIPTION
**Pull request by tomas-dora**

### CODE REVIEW

File: testnets/deardogetestnet/chain.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "deardogetestnet",
```
Comment (JeremyParish69) [inline testnets/deardogetestnet/chain.json:3]: please rename chain_name (in both files) and the directory to doravotatestnet.

File: testnets/deardogetestnet/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      "base": "peaka",
+      "name": "Dear Doge Testnet",
+      "display": "DORA",
+      "symbol": "DORA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/deardogetestnet/images/deardoge.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/deardogetestnet/images/deardoge.png"
```
Comment (JeremyParish69) [inline testnets/deardogetestnet/assetlist.json:26]: A duplicate image is not needed. Please point this to Doravota.  An image_sync would be best.

File: testnets/deardogetestnet/chain.json (hunk 1 above)
Comment (tomas-dora) [reply testnets/deardogetestnet/chain.json:3]: These are two different testnets, so we need to keep both deardogetestnet and doravotatestnet

File: testnets/deardogetestnet/assetlist.json (hunk 2 above)
Comment (tomas-dora) [reply testnets/deardogetestnet/assetlist.json:26]: deardoge's logo may change after that.

File: testnets/deardogetestnet/chain.json (hunk 1 above)
Comment (JeremyParish69) [reply testnets/deardogetestnet/chain.json:3]: Sorry, I'd forgotten there was already a doravotatestnet when I commented. But I presume it's still a testnet for doravota, so I think in this case it should be doravotatestnet2, a convention we see with many other chains with multiple testnets

**Comment by tomas-dora** [comment]
@JeremyParish69 hi, Our new test network is up and running, I've updated the latest configuration info, please help merge this pr. Thanks!

**Comment by tomas-dora** [comment]
@JeremyParish69  I've updated some more detail data and now it's ready, can you merge this pr?

File: testnets/deardogetestnet/chain.json (hunk 1 above)
Comment (JeremyParish69) [reply testnets/deardogetestnet/chain.json:3]: Sorry, I don't see how this is resolved. The git_repo and genesis_url still point to dora vota, so it still looks like this should be doravota2...

Comment (tomas-dora) [reply testnets/deardogetestnet/chain.json:3]: I fixed this problem. @JeremyParish69 
https://github.com/cosmos/chain-registry/pull/3213/files#diff-14bbdcbd15770599c05050af74677168f99c7bf946a103deaf11139e23d365e3R29-R34

**Pull request merged**